incubator/mysqlha/templates/statefulset.yaml

```diff
@@ -0,0 +1,249 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  serviceName: {{ template "fullname" . }}
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[
+          {
+            "name": "init-mysql",
+            "image": "mysql:{{ .Values.imageTag }}",
+            "command": ["bash", "-c", "
+              set -ex\n
```

Comment:
Since there is a lot going on in here can you split this out into its own file then use a Files.Get+ indent to add it in here?
There is an example in the spinnaker chart:
https://github.com/kubernetes/charts/blob/master/stable/spinnaker/templates/configmap/spinnaker-config.yaml#L11